tests/integration/docker/Dockerfiles/README.md

```diff
@@ -0,0 +1,17 @@
+# NGINX image for Intel® Clear Containers swarm tests
+
+In order to build the nginx image which is used in our swarm tests,
+follow these steps:
+
+1. Building the nginx image with the following command:
```

Comment:
nit: replace Building with Build
Build the nginx image with the following command: